src/components/SuccessPage/SuccessPage.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import Link from 'next/link'
+
+const SuccessPage = ({ workOrderReference, variationApproved }) => {
+  const text = variationApproved
```

Comment:
Should we just pass all this text as props?

So we can reuse this component for the RCC agent authorisation flow

Reply:
I'm not sure this component needs to know whether a variation was approved or not - that can come from the parent. Then similar for when we have an authorisation approved